data/companies.yml

```diff
@@ -434,3 +434,6 @@ users:
   - name: "Zendesk"
     logo: "/logos/zendesk.svg"
     url: "https://www.zendesk.com/"
+  - name: "Trendyol"
+    logo: "/logos/trendyol.png"
+    url: "https://www.trendyol.com/"
```

Comment:
Note to self: I should update the code to sort the names alphabetically
Note to you: could you please revise the PR to sort your name alphabetically? 🙂

Reply:
Thanks, I changed it can you check? :slightly_smiling_face: